.travis.yml

```diff
@@ -183,7 +182,14 @@ jobs:
         - ./tests/scripts/start-compose.sh ./tests/environment/docker-compose-all-amps-test.yml
         - ./tests/scripts/wait-for-alfresco-start.sh "http://localhost:8082/alfresco"
         - travis_wait 20 mvn install -f tests/tas-all-amps/pom.xml -Pall-tas-tests -Denvironment=default -DrunBugs=false
-
+    - name: "Veracode static code scan"
```

Comment:
Similar to WhiteSource scan, we could run this only on master, support branches and on builds != pull_requests:
`     if: fork = false AND (branch = master OR branch =~ /support\/.*/) AND type != pull_request`.

Reply:
The change has been made.